Tests/Functional/Drivers/Queue/StompQueueDriverTest.php

```diff
@@ -86,7 +86,7 @@ public function testShouldSendReceiveAndAckOnce(MessageInterface $msg)
 
         $this->driver->ack();
 
-        sleep(1);
+        \sleep(1);
```

Comment:
This should have a special group, see: https://symfony.com/doc/master/components/phpunit_bridge.html#clock-mocking

Reply:
I think in this case we actaully do want to sleep for 1 sec, rather then mock sleep somewhere else

Reply:
In that case, ok.